ws-nextjs-app/pages/[service]/av-embeds/AvEmbedsMetadata.tsx

```diff
@@ -0,0 +1,88 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { Helmet } from 'react-helmet';
+import filterForBlockType from '../../../../src/app/lib/utilities/blockHandlers';
+import buildAvEmbedURL from '../../../utilities/buildAvEmbedUrl';
+import { AvEmbedsPageProps } from './types';
+
+const AvEmbedsMetadata = ({ pageData }: AvEmbedsPageProps) => {
+  const { mediaBlock, metadata, promo } = pageData;
+  const promoSummary =
+    promo.summary.blocks[0].model.blocks[0].model.blocks[0].model.text;
+  const headline = promo.headlines.seoHeadline;
+  const { assetId, language, mediaDelimiter, mediaId, service, variant } =
+    metadata;
+  const aresMediaBlock = filterForBlockType(mediaBlock, 'aresMedia');
+  const aresMediaMetadata = filterForBlockType(
+    aresMediaBlock.model.blocks,
+    'aresMediaMetadata',
+  );
+  const captionBlock = filterForBlockType(
+    aresMediaBlock.model.blocks,
+    'captionText',
+  );
+  const { imageUrl } = aresMediaMetadata.model;
+  const { caption } = captionBlock.model;
+  const mediaURL = buildAvEmbedURL({
+    assetId,
+    mediaDelimiter,
+    mediaId,
+    service,
+    variant,
+  });
+
+  return (
+    <Helmet>
+      <meta
+        name="viewport"
+        content="width=device-width, initial-scale=1, user-scalable=1"
+      />
+      <meta charSet="utf-8" />
+      <meta httpEquiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+      <meta name="description" content={promoSummary} />
+
+      <link
+        href="//ichef.bbci.co.uk"
+        rel="preconnect"
+        crossOrigin="anonymous"
+      />
+
+      <link rel="dns-prefetch" href="//static.bbci.co.uk" />
+      <link rel="dns-prefetch" href="//static.bbc.co.uk" />
```

Comment:
Should this be static.files (or are we taking this from the existing metadata for syndicated AV Embeds?

Reply:
This too was lifted from the Morph implementation, but may not be required?

Reply:
Might be best / easiest to leave the values as per Morph, just in case they are required?